pages/common/views/pdf/header.jsx

```diff
@@ -17,7 +17,7 @@ const Header = ({ model, licenceType, nonce }) => (
 
         <Fragment> | Granted: <strong>{format(model.issueDate)}</strong></Fragment>
 
-        { licenceType === 'pil' &&
+        { ['pil', 'pel'].includes(licenceType) &&
           <Fragment> | Last amended: <strong>{format(model.updatedAt)}</strong></Fragment>
```

Comment:
This will likely need to be updated somewhere to reflect the fact that roles, authorisations and places are also part of the PEL, and so just taking the `updated_at` from the establishments table is not sufficient to provide a "Last amended" on the pEL itself.

Probably do that in the resolver as another ticket though.

Reply:
was a ticket added for this?